Content.Server/StationEvents/Components/CommunicationInterceptionRuleComponent.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Content.Server.StationEvents.Events;
+using Content.Server.AlertLevel;
+
+namespace Content.Server.StationEvents.Components;
+
+[RegisterComponent, Access(typeof(CommunicationInterceptionRule))]
+public sealed partial class CommunicationInterceptionRuleComponent : Component
+{
+    /// <summary>
+    /// Alert level to set the station to when the event starts.
+    /// </summary>
+
```

Comment:
remove readonly and add [DataField]
also im pretty sure this can use ProtoId<AlertLevelPrototype> or something